doc/api/stream.md

```diff
@@ -1505,24 +1713,25 @@ var writable = new stream.Writable({
 
 <!--type=misc-->
 
-Both Writable and Readable streams will buffer data on an internal
-object which can be retrieved from `_writableState.getBuffer()` or
-`_readableState.buffer`, respectively.
+Both [Writable][] and [Readable][] streams will buffer data on an internal
+object which can be retrieved using `writable._writableState.getBuffer()` or
+`readable._readableState.buffer`, respectively.
 
-The amount of data that will potentially be buffered depends on the
-`highWaterMark` option which is passed into the constructor.
+The amount of data potentially be buffered depends on the `highWaterMark` option
+which is passed into the streams constructor.
 
-Buffering in Readable streams happens when the implementation calls
+Data is buffered in Readable streams when the implementation calls
 [`stream.push(chunk)`][stream-push]. If the consumer of the Stream does not
-call [`stream.read()`][stream-read], then the data will sit in the internal
+call [`stream.read()`][stream-read], the data will sit in the internal
 queue until it is consumed.
 
-Buffering in Writable streams happens when the user calls
-[`stream.write(chunk)`][stream-write] repeatedly, even when it returns `false`.
+Data is buffered in Writable streams happens when the
+[`stream.write(chunk)`][stream-write] method is called repeatedly, even if calls
+return `false`.
```

Comment:
It might be more surprising that a writable could be buffering even though it's returning `true` (vs. `false`!)